### ISSUE TITLE
[L] BCeID setup

### ISSUE THREAD

**Issue by Dianadec** [bcgov/reserve-rec-api#32]
#### Description:

Next steps for BCeID integration. 

#### Acceptance Criteria:
As a public park user
When I go to log in to the system to book a camping reservation
Then I see a button that allows me to login with my BCeID account.

#### Development Checklist:

- [ ] pre-req: Cognito setup
- [ ] Set up in dev and test environments
- [ ] Login page shows option to click BC Services card like the following:
![Image](https://github.com/user-attachments/assets/6d83407e-9ae1-4131-b356-3b89c8f7370c)

#### Dependencies

- Blocked by https://github.com/bcgov/reserve-rec-api/issues/36
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- Linked to ticket https://github.com/orgs/bcgov/projects/49/views/2?sliceBy%5Bvalue%5D=Reservations&pane=issue&itemId=84717072&issue=bcgov%7Creserve-rec-api%7C34
-

**Comment by LindsayMacfarlane** [comment]
Hi @marklise - please help refine this ticket :)

**Comment by Dianadec** [comment]
Whoever pulls in this ticket, talk to Mark about the deets.

**Comment by Dianadec** [comment]
**Sprint Planning Notes**
Questions:
- do we even need this for admins? (LM: Yes - for POs)
- are we going to want this as a login for public? (Yes, until we have social login stuff setup. People outside of BC will need to login ( - they can't use BC Services Card)).
Context:
- Some POs needed BCeID so that's why we used it for DUP.

**Comment by Dianadec** [comment]
Public side and admin side are two different identities (diff in keyloack realm or whatever realm we're working with).

May need to split this.

**Comment by meredithom** [comment]
Duplicate of https://github.com/bcgov/reserve-rec-admin/issues/81
